api/clever/__init__.py

```diff
@@ -56,8 +59,18 @@ class CleverAuthenticationAPI(OAuthAuthenticator):
     # need to get a code from First Book instead.
     SUPPORTED_USER_TYPES = ['student', 'teacher']
 
+    TOKEN_TYPE = "Clever token"
+
     log = logging.getLogger('Clever authentication API')
 
+    def _data_source(self, _db):
+        data_source, is_new = get_one_or_create(
+            _db, DataSource, name="Clever",
+            offers_licenses=False,
+            primary_identifier_type=None
+        )
```

Comment:
This might be the first data source that's not in the well-known DataSources. I don't think it should be there - nothing outside this module should be using it and we don't want to have to change core for every new authentication provider.